extensions/authclient/BaseOAuth.php

```diff
@@ -397,7 +397,7 @@ protected function composeUrl($url, array $params = [])
     /**
      * Saves token as persistent state.
      * @param OAuthToken $token auth token
-     * @return static self reference.
+     * @return $this
```

Comment:
the return value should have a comment so one can read what is returned even if you do not know what `return $this` means.